postgreCDC TO hive

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

Caused by: org.apache.seatunnel.engine.common.exception.SeaTunnelEngineException: java.lang.NullPointerException
at org.apache.seatunnel.connectors.cdc.debezium.row.SeaTunnelRowDebeziumDeserializationConverters.convert(SeaTunnelRowDebeziumDeserializationConverters.java:88)

### SeaTunnel Version

2.3.4

### SeaTunnel Config

```conf
env {
execution.parallelism = 1
job.mode = "STREAMING"
checkpoint.interval = 15000
}

source {
Postgres-CDC {
result_table_name ="fake"
base-url="jdbc:postgresql:///test"
source.reader.close.timeout = 120000
username = ""
password = ""
driver = "org.postgresql.Driver"
slot.name = "seatunnel1716623306008"
database-names = [""]
schema-names = [""]
table-names = [".."]
}
}

transform {
Sql{
source_table_name = "fake"
query = "select id,name,create_time,'$[yyyy-MM-dd]' as pt from fake "
result_table_name = "fake1"
}
FieldMapper {
source_table_name = "fake1"
result_table_name = "fake2"
field_mapper = {
id=id
name=name
create_time=create_time
pt = pt
}
}
}

sink {
Hive {
source_table_name = "fake2"
table_name ="."


}
}
```


### Running Command

```shell
env {
execution.parallelism = 1
job.mode = "STREAMING"
checkpoint.interval = 15000
}

source {
Postgres-CDC {
result_table_name ="fake"
base-url="jdbc:postgresql:///test"
source.reader.close.timeout = 120000
username = ""
password = ""
driver = "org.postgresql.Driver"
slot.name = "seatunnel1716623306008"
database-names = [""]
schema-names = [""]
table-names = [".."]
}
}

transform {
Sql{
source_table_name = "fake"
query = "select id,name,create_time,'$[yyyy-MM-dd]' as pt from fake "
result_table_name = "fake1"
}
FieldMapper {
source_table_name = "fake1"
result_table_name = "fake2"
field_mapper = {
id=id
name=name
create_time=create_time
pt = pt
}
}
}

sink {
Hive {
source_table_name = "fake2"
table_name ="."


}
}
```


### Error Exception

```log
postGreCDC  to  hive:when job start, source  data insert or delete is ok,,but once source data updated  like update table set name ='name2' where id =2 ,it will appear
Caused by: org.apache.seatunnel.engine.common.exception.SeaTunnelEngineException: java.lang.NullPointerException
at org.apache.seatunnel.connectors.cdc.debezium.row.SeaTunnelRowDebeziumDeserializationConverters.convert(SeaTunnelRowDebeziumDeserializationConverters.java:88)
```


### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java
+++ b/seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java
@@ -184,7 +184,7 @@ public final class SeaTunnelRowDebeziumDeserializeSchema
             delete.setRowKind(RowKind.DELETE);
             delete.setTableId(tableId);
             collector.collect(delete);
-        } else {
+        } else if (operation == Envelope.Operation.UPDATE) {
             SeaTunnelRow before = extractBeforeRow(converters, record, messageStruct, valueSchema);
             before.setRowKind(RowKind.UPDATE_BEFORE);
             before.setTableId(tableId);

--- a/seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java
+++ b/seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java
@@ -194,6 +194,8 @@ public final class SeaTunnelRowDebeziumDeserializeSchema
             after.setRowKind(RowKind.UPDATE_AFTER);
             after.setTableId(tableId);
             collector.collect(after);
+        } else {
+            log.warn("Received {} operation, skip", operation);
         }
     }
 

